packages/web/src/open-feature.ts

```diff
@@ -393,7 +393,7 @@ export class OpenFeatureAPI
         const maybePromise = wrapper.provider.onContextChange(oldContext, newContext);
 
         // only reconcile if the onContextChange method returns a promise
-        if (typeof maybePromise?.then === 'function') {
+        if (typeof (maybePromise as Promise<void>)?.then === 'function') {
```

Comment:
This is the only thing the ng-compiler was complaining about I could not ignore - obviously there's no runtime change here, it just makes the compiler happy.